core/unit_test/incremental/Test06_ParallelFor_MDRangePolicy.hpp

```diff
@@ -52,34 +52,35 @@
 // elements as a product of iterator indexes and a constant. At the end, we
 // check for correctness.
 
-namespace Test04 {
+namespace Test06 {
 
-using value_type       = double;
-const int N            = 10;
-const int M            = 10;
-const value_type delta = 0.5;
+using value_type = double;
+const int N      = 10;
+const int M      = 10;
```

Comment:
Same thing as above. Either change all of them to `constexpr` or make all of them data members, but don't alternate between the two techniques unless there's a good reason to do so.